OrbitLinuxTracing/TracerThread.h

```diff
@@ -43,6 +54,28 @@ class TracerThread {
   void Run(const std::shared_ptr<std::atomic<bool>>& exit_requested);
 
  private:
+  void ProcessContextSwitchEvent(const perf_event_header& header,
```

Comment:
Per the style guide, PerfEventRingBuffer should be passed as a pointer as it's not a const reference.